libcontainer/configs/mount.go

```diff
@@ -3,7 +3,7 @@ package configs
 const (
 	// EXT_COPYUP is a directive to copy up the contents of a directory when
 	// a tmpfs is mounted over it.
-	EXT_COPYUP = 1 << iota
+	EXT_COPYUP = 1 << iota //nolint:golint // ignore don't use ALL_CAPS in Go names; use CamelCase (golint)
```

Comment:
Perhaps
```suggestion
	EXT_COPYUP = 1 << iota //nolint:golint // ignore "don't use ALL_CAPS in Go names; use CamelCase (golint)"
```

or just
```suggestion
	EXT_COPYUP = 1 << iota //nolint:golint // ignore "don't use ALL_CAPS" warning
```